localnet/poktrolld/config/relayminer_beta_config.yml

```diff
@@ -0,0 +1,99 @@
+# TODO_CONSIDERATION: We don't need this now, but it would be beneficial if the logic handling this config file could be designed in such a way that it allows for "soft" config changes in the future, meaning changes without restarting a process.
+#                     This would be useful for adding a proxy or a supplier without interrupting the service.
+
+pocket:
+  # URL of the Pocket node that will be used to query the network.
+  query_node_url: tcp://localhost:36657
+  # URL of the Pocket node that will be used to submit transactions.
+  network_node_url: tcp://127.0.0.1:36657
+
+# Name of the key from the keyring that will be used to sign messages.
+# The key should be the same key that is staked for this RelayMiner.
+signing_key_name: servicer1
+
+# Proxies are endpoints that expose different suppliers to the internet.
+# 
+# TODO_CONSIDERATION: if there are 0 suppliers pointing to the proxy defined in that map, don't enable the port BUT log a warning that the endpoint is not used so the port is not exposed.
+# TODO_CONSIDERATION: if a supplier is referencing a proxy that has a different type, throw an error. (this should make it easier to avoid confusion/potential configuration errors if an unsupported proxy is referenced) 
+# TODO_DISCUSS: I changed the map for an array, with each element having a "name" field. That way, it is evident what we proxy are referencing by name inside of the supplier.
+proxies:
+    # Name of the proxy. It will be used to reference in a supplier. Must be unique.
+    # Required.
+    # 
+    # TODO_CONSIDERATION: if we enforce DNS compliant names, it can potentially become handy in the future. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names. Not necessary though.
+  - name: http-example
+    # Hostname to open port on. Use 0.0.0.0 in containerized environments, 127.0.0.1 with a reverse-proxy when there's another process on localhost that can be used as a reverse proxy (nginx, apache, traefik, etc.).
+    # Required (at least for http, ws, tcp?).
```

Comment:
This is a subject I want us to discuss as to what kind of servers (listeners) we should support. I see no more than (tcp, http, https, unix and udp) in this section, all the others (ws, ws, REST, gRPC, JsonRPC...) are remote procedure call mechanisms that may use the former as transport medium.

Obviously http and https function over tcp and are not transport protocols in the OSI sense but since the industry is (ab)using them as such, we can have them in the transport team.

The question we should ask to help categorizing them is: Are we going to run a server/listener for them or not?
The answer is **yes** for http, https, tcp, unix, udp, ftp...
The answer is **no** for:
* Websocket: which runs on http(s) with `Upgrade: websocket`, `Connection: Upgrade` and `Sec-Websocket-Key: ...` http headers
* gRPC: which runs on http(s) with `Content-Type: application/grpc` http header but can also run over raw tcp too
* JsonRPC: http(s) with `Content-Type: application/json` http header and `{jsonrpc: 2.0, id: ...}` payload
* REST: http(s) with extensive usage of http verbs/methods (obviously http)

Currently this isn't a problem, since we'll mainly use http(s) listeners/servers. But it is important to have a clear categorization and vision on what each of these is and how we're going to leverage it.

cc @Olshansk , @bryanchriswhite